core/server/utils/markdown-converter.js

```diff
@@ -7,16 +7,36 @@ var MarkdownIt  = require('markdown-it'),
     .use(require('markdown-it-footnote'))
     .use(require('markdown-it-lazy-headers'))
     .use(require('markdown-it-mark'))
-    .use(require('markdown-it-named-headers'), {
-        // match legacy Showdown IDs otherwise default is github style dasherized
-        slugify: function (inputString, usedHeaders) {
+    .use(function namedHeaders(md) {
+        // match legacy Showdown IDs
+        var slugify = function (inputString, usedHeaders) {
             var slug = inputString.replace(/[^\w]/g, '').toLowerCase();
             if (usedHeaders[slug]) {
                 usedHeaders[slug] += 1;
                 slug += usedHeaders[slug];
             }
             return slug;
-        }
+        };
+        var originalHeadingOpen = md.renderer.rules.heading_open;
+
+        md.renderer.rules.heading_open = function (tokens, idx, something, somethingelse, self) {
```

Comment:
Could you please add the same comment as you've added in the Ghost-Admin PR? This code was copied and has no tests.

https://github.com/TryGhost/Ghost-Admin/pull/856/files#diff-4590e8e15d1359792e9fb142d668c2cbR17

Reply:
👍 done